api/envoy/extensions/filters/network/redis_proxy/v3/redis_proxy.proto

```diff
@@ -127,6 +127,10 @@ message RedisProxy {
 
     // Read policy. The default is to read from the primary.
     ReadPolicy read_policy = 7 [(validate.rules).enum = {defined_only: true}];
+
+    bool redis_cx_rate_limit_enabled = 10;
```

Comment:
These fields are not documented, can you please add comments explaining this feature, and there are a couple of changes I'd suggest:
* Don't prefix with `redis_`, since the context implies that.
* Add a new `ConnectionRateLimit` submessage to the parent message and a field `ConnectionRateLimit rate_limit = 10;` instead. There can be a `connection_rate_limit_per_sec` field there. The presence of the message field can be used to determine if enabled or not.

You will also need to update version history. Thanks.